ciri/conf.c

```diff
@@ -137,7 +137,7 @@ static retcode_t set_conf_value(iota_ciri_conf_t* const ciri_conf,
       api_conf->max_get_trytes = atoi(value);
       break;
     case 'p':  // --port
-      api_conf->port = atoi(value);
+      // api_conf->port = atoi(value);
```

Comment:
Why is the port not an  option anymore ?

Reply:
the port is purely HTTP specific. Adding a HTTP config object is still on the todo list to replace this.

Reply:
👍